assignments/stretch-function-conversion.js

```diff
@@ -2,22 +2,50 @@
 
 // let myFunction = function () {};
 
+let myFunction = () => {
+
+}
+
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = param => {
+	return param;
+}
+
```

Comment:
Can be refactored to:
`let anotherFunction = param => param`
- `return` keyword is implicit when your arrow function has 1 line